world-server/src/world/World.ts

```diff
@@ -21,29 +24,99 @@ export class World {
     public gameTime: number = 0;
     // List of players sorted by descending score
     public scoreboard: string[] = [];
-    public gameSeed: number = 0;
+    public gameSeed: string = "";
 
-    constructor(gameTime: number){
+    // Used to control the frequency of the simplex noise
+    private terrainFrequency = 0.1;
+
+    constructor(gameTime: number) {
         this.gameTime = gameTime
     }
 
     init() {
         // Generate random game seed upon initialization
         let randBytes = randomBytes(4);
-        this.gameSeed = randBytes.readUInt32BE();
-    
+        this.gameSeed = randBytes.toString();
+
         this.generateWorld();
         this.populateShopItems();
 
     }
 
-    generateWorld(){
-        // TODO: generate world :)
+    generateWorld() {
+        /* var simplex = new SimplexNoise(this.gameSeed), value2d = simplex.noise2D(10, 10); */
+        // gre skozi vsa polja
+        for (let i = 0; i < 65; i++) {
```

Comment:
Copy & paste kode iz Backend na Frontend nikoli ni OK :). Tukaj (mentorji) pogledamo, da bi vrgli v world-core in bo na voljo na obeh straneh,